content/en/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,390 @@
+---
+reviewers:
+- sftim
+title: LimitRange
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on kubernetes cluster.
+With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
+Within a namespace, a Pod/Container can still consume as much CPU and memory as the defined namespace's resource quota.
+there is a concern that one Pod/Container could monopolise its namespace resources.
+LimitRange is a tool to address this concern.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+A limit range, defined by a `LimitRange` object, provides constraints that can :
+- Enforce min/max compute resources usage per Pod/Container in a namespace
+- Enforce min/max storage request per PersistentVolumeClaim in a namespace

```

Comment:
- please remove space after `can`, and add a new line to fix the list?
- spell out: Enforce minimum and maximum ... per Pod or Container

Reply:
Thanks Fixed